job-service-postgres-container/scripts/setup.sh

```diff
@@ -19,9 +19,9 @@
 echo "Installing Job Service database."
 
 ./install_job_service_db.sh \
-  -db.user $POSTGRES_USER \
-  -db.pass $POSTGRES_PASSWORD \
+  -db.user ${POSTGRES_USER} \
+  -db.pass ${POSTGRES_PASSWORD} \
   -db.name ${POSTGRES_DB:-jobservice} \
-  -db.connection jdbc:postgresql://127.0.0.1:5432
+  -db.connection jdbc:postgresql://127.0.0.1:5432/
```

Comment:
Yes, to match the expected patter in the flyway utility as we expect the format `jdbc:postgresql://localhost:3307/` 
Otherwise, I can update the utility to update the url when required